Makefile

```diff
@@ -8,22 +8,22 @@ docs-clean:
 
 # Avoid conflicts between .pyc/pycache related files created by local Python interpreters and other interpreters in Docker
 python-caches-clean:
-	-find ./ -name "__pycache__" -exec rm -rf -- \{\} \;
-	-find ./ -name ".pyc" -exec rm -rf -- \{\} \;
+	-@find . -name "__pycache__" -exec rm -rf -- \{\} \;
+	-@find . -name ".pyc" -exec rm -rf -- \{\} \;
 
 docs:
 	cd docs && $(MAKE) html
 
 test:
 	python3 setup.py test
 
-it:
+it: python-caches-clean
 	tox
 
-it34:
+it34: python-caches-clean
 	tox -e py34
 
-it35:
+it35: python-caches-clean
 	tox -e py35
 
 it36:
```

Comment:
Should also need `python-caches-clean`?

Reply:
Thanks forgot this one! Addressed it now.